stats/view/view.go

```diff
@@ -95,7 +95,7 @@ func (v *View) canonicalize() error {
 	})
 	sort.Float64s(v.Aggregation.Buckets)
 	for _, b := range v.Aggregation.Buckets {
-		if b < 0 {
+		if b <= 0 {
 			return ErrNegativeBucketBounds
```

Comment:
This should be renamed "ErrNonPositiveBucketBounds" now.

Reply:
done.

Reply:
I changed it back as I am silently dropping 0 bound bucket.

Reply:
OK, silently dropping it makes more sense.